addons/mail/static/tests/tours/dynamic_placeholder_tour.js

```diff
@@ -0,0 +1,131 @@
+                ) {
+                    console.error(`Email template did not show correct notification.`);
+                }
+            },
+        },
+        {
+            content: 'Select "Contact" model',
+            trigger: 'div[name="model_id"] input[type="text"]',
+            run: "text Contact",
+        },
+        {
+            content: "Wait for model to load and click on contact",
+            trigger: 'div[name="model_id"] .ui-autocomplete .dropdown-item:not(:has(.fa-spin))',
+            run: async function () {
+                const contact = Array.from(
+                    document.querySelectorAll(
+                        'div[name="model_id"] .ui-autocomplete .dropdown-item'
+                    )
+                ).find((el) => el.textContent === "Contact");
+                await contact.click();
+            },
+        },
+        {
+            content: "Wait for the drop down to disappear",
+            trigger: 'div[name="model_id"] .o-autocomplete:not(:has(.ui-autocomplete))',
+            run: function () {},
+        },
+        {
+            content: 'Retry insert # inside "Subject" input',
+            trigger: 'div[name="subject"] input[type="text"]',
+            run(actions) {
+                actions.text(`yes_model_id #`, this.$anchor);
+                this.$anchor[0].dispatchEvent(
+                    new KeyboardEvent("keydown", { bubbles: true, key: "#" })
+                );
+            },
+        },
+        {
+            content: "Check if the dynamic placeholder popover is opened",
+            trigger: "div.o_field_selector_popover",
+            run: function () {},
+        },
+        {
+            content: "Click on the first entry of the dynamic placeholder",
+            trigger: "div.o_field_selector_popover li:first-child",
+        },
+        {
+            content: "Enter a default value",
+            trigger:
+                'div.o_field_selector_popover .o_field_selector_default_value_input input[type="text"]',
+            run: "text defValue",
+        },
+        {
+            content: "Click on the the dynamic placeholder default value",
+            trigger: "div.o_field_selector_popover li:first-child",
+        },
+        {
+            content: "Wait for the popover to disappear",
+            trigger: "div.o_popover_container:empty",
+            run: function () {},
+        },
+        {
+            content: "Check if subject value was correclty updated",
+            trigger: 'div[name="subject"] input[type="text"]',
+            run() {
+                const subjectValue = this.$anchor[0].value;
+                const correctValue =
+                    "yes_model_id {{object.activity_exception_decoration or '''defValue'''}}";
+                if (subjectValue !== correctValue) {
+                    console.error(
+                        `Email template have "${correctValue}" in subject input (actual: ${subjectValue})`
```

Comment:
I don't understand this sentence. Isn't it supposed to be like "Email template _should_ have ..." instead ?

Reply:
yes indeed

Reply:
I'll change the sentence